### DISCUSSION TITLE
Collapse plugin

### DISCUSSION THREAD

**Discussion by hakankaraduman** [remarkjs/remark#628] [Q&A] [answered]
Hi,

I use react-markdown and I need details-summary elements.

How can I make this plugin "https://github.com/Rokt33r/remark-collapse" work with it?

I also need it multiple times in the markdown file, is there a solution to this requirement? Thanks in advance.

**Comment by ChristianMurphy** [comment, marked as answer]
It's likely related to https://github.com/remarkjs/react-markdown/issues/384 and/or https://github.com/remarkjs/react-markdown/issues/427
Currently `react-markdown` doesn't handle html in markdown the same as other remark based solutions.
https://github.com/remarkjs/react-remark could be used as a work around, as it supports `hProperties` and `html` through `rehype`.
Or you could wait for or help out on resolving https://github.com/remarkjs/react-markdown/issues/427
